packages/graph/hash_graph/bench/benches/util.rs

```diff
@@ -0,0 +1,195 @@
+use std::{mem::ManuallyDrop, str::FromStr};
+
+use graph::{
+    ontology::AccountId,
+    store::{
+        AsClient, DataTypeStore, DatabaseConnectionInfo, DatabaseType, EntityTypeStore,
+        LinkTypeStore, PostgresStorePool, PropertyTypeStore, StorePool,
+    },
+};
+use tokio::runtime::Runtime;
+use tokio_postgres::NoTls;
+use type_system::{DataType, EntityType, LinkType, PropertyType};
+
+type Pool = PostgresStorePool<NoTls>;
+pub type Store = <Pool as StorePool>::Store<'static>;
+
+// TODO - deduplicate with integration/postgres/mod.rs
```

Comment:
This was mentioned in the PR description, will create and attach an asana task if people agree with it